templates/add_movie.html

```diff
@@ -87,18 +87,40 @@
                     <img src="{img_url}" class="card-img-top">
                     <div class='btn-group'>
                         <div class="dropdown w-75">
-                            <button type="button" class="btn btn-secondary dropdown-toggle w-100 rounded-0" data-toggle="dropdown">
+                            <button type="button" class="btn btn-secondary dropdown-toggle w-100 rounded-0" onclick="add_movie_settings(event, this)">
```

Comment:
This button now displays 2 selectors and another add button.
I don't really like too much, add-movie-form is displayed with slideDown, probably it would be better as float div (like dropdown is open), but I'm thinking it may be better to remove this add button and have a big show details button.